.github/workflows/trigger-e2e-environment.yml

```diff
@@ -23,9 +23,52 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  gathering_information:
```

Comment:
We often tend to name things in a "command form" so "gather_information" would be more aligned with that. You can also name this job as `generate_stack_name` or similar as it's effectively what the outcome of this job is

Reply:
Ok